src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -274,9 +247,14 @@ private ImmutableArray<IOperation> GetIOperationChildren(BoundNode boundNode)
             }
 
             var builder = ArrayBuilder<IOperation>.GetInstance(boundNodeWithChildren.Children.Length);
-            foreach (var childNode in boundNodeWithChildren.Children)
+            foreach (BoundNode childNode in boundNodeWithChildren.Children)
```

Comment:
Are there implementations of `Children` that allocate? If so, extract a local for `boundNodeWithChildren.Children` to avoid unnecessary allocations.

Reply:
ah, I got it. okay.